src/main/java/org/gridsuite/modification/server/modifications/VscModification.java

```diff
@@ -257,6 +265,11 @@ private List<ReportNode> hvdcAngleDroopActivePowerControlAdder(HvdcLine hvdcLine
         return reports;
     }
 
+    private static NetworkModificationException modifyHvdcAngleDroopActivePowerControl(@lombok.NonNull String equipementName, @lombok.NonNull String attributeName) {
```

Comment:
Not necessary a static method..

Reply:
The method creates an exception, should rename as createWrongAngleDroopActivePowerControlException for example